core/src/com/unciv/logic/automation/NextTurnAutomation.kt

```diff
@@ -815,7 +815,7 @@ object NextTurnAutomation {
             when {
                 unit.baseUnit.isRanged() -> rangedUnits.add(unit)
                 unit.baseUnit.isMelee() -> meleeUnits.add(unit)
-                unit.hasUnique("Bonus for units in 2 tile radius 15%")
+                unit.getMatchingUniques(UniqueType.GreatPerson).any { it.params[0] == "War" }
```

Comment:
... and treating one unique as detector for the support of the effects of another unique?

Reply:
> treating one unique as detector for the support of the effects of another unique?

Not sure I understand what you mean. Having 2 uniques linked together by a "hidden" logic does not sound like a good idea for me.

Reply:
Exactly. You look for 'Great Person - War' to decide whether the 'Bonus in Radius' code should run. Forgot the details - there are two such places? One decides whether the automation runs early or late - here late is done to have the bonus "field" follow the military, which is a function of the other Unique. The other place was even a bit more directly linked from hazy memory...

Reply:
> You look for 'Great Person - War' to decide whether the 'Bonus in Radius' code should run.

I don't like this approach since "Bonus in Radius" may belong to any other modder unit, not only Great Person, as well as custom Great Persons - War may do not have this "aura". Let's do not mess them up together.

Reply:
Then I must be befuddled. Seemed to me that is exactly what your code does. :shrug: